terraform/serviceaccount.tf

```diff
@@ -0,0 +1,12 @@
+module "google_service_account" {
+  source        = "terraform-google-modules/service-accounts/google"
+  version       = "~> 4.1.1"
+  project_id    = "${var.project_id}"
+  prefix        = "${var.common_name}"
+  names         = ["worker-svc"]
+  project_roles = [
+    "${var.project_id}=>roles/bigtable.admin",                   # Generate table within Bigtable instance
+    "${var.project_id}=>roles/storage.objectCreator",            # Generate components/edge bucket objects
```

Comment:
What is the purpose of this service account? If its for creating the infrastructure, it will not need gcs permissions.

The service account key with gcs access goes into helm values file, which workers will need to create objects.
Also they will need `roles/storage.objectAdmin` because creator doesn't have permission to delete/replace a file when re-trying a failed job.